### PR TITLE
DOC-532: Update and correct errors in copyrights page

### PR DESCRIPTION
**Pull request by lloadholt**
Updated the copyrights page.

### CODE REVIEW

File: docs/copyright_license.rst [hunk 1]
```diff
@@ -24,22 +24,23 @@ Use #4040, included at the top level of the `Connector for JavaScript repository
...
+solely under and subject to RTI’s standard terms and conditions available 
+at https://www.rti.com/terms and in accordance with your License 
+Acknowledgement Certificate (LAC) and Maintenance and Support Certificate 
+(MSC), except to the extent otherwise agreed to in writing by RTI.
+
 
-This is an independent publication and is neither affiliated with, nor
-authorized, sponsored, or approved by, Microsoft Corporation.
 
 The security features of this product include software developed by the
```
Comment (lloadholt) [inline docs/copyright_license.rst:45]: @Alxe Can you confirm if this paragraph about security is needed?

Comment (rkorte) [reply docs/copyright_license.rst:45]: @lloadholt, changes to Connector documentation need to happen in support/connector/1.3.0 and develop. I think release/connector/1.3.1 is frozen, so improvements should be made to support or develop. @Alxe, please confirm.

Comment (Alxe) [reply docs/copyright_license.rst:45]: @rkorte is correct. Release 1.3.1 has shipped so this release branch is effectively frozen.

Please retarget this branch to either `develop` or `support/connector/1.3.0`. Once this has been merged, you need to cherry-pick the changes into the other branch.

Comment (lloadholt) [reply docs/copyright_license.rst:45]: Thanks @Alxe . After updating ``support/connector/1.3.0``, do I cherry-pick to ``develop``?

Comment (rkorte) [reply docs/copyright_license.rst:45]: I think yes, @lloadholt

**Pull request closed without merging**